pkg/types/machinepools.go

```diff
@@ -20,6 +20,8 @@ const (
 	MachinePoolEdgeRoleName = "edge"
 	// MachinePoolControlPlaneRoleName name associated with the control plane machinepool.
 	MachinePoolControlPlaneRoleName = "master"
+	// MachinePoolArbiterRoleName name associated with the control plane machinepool for smaller sized limited nodes.
+	MachinePoolArbiterRoleName = "arbiter"
```

Comment:
Need to update the comments on ll. 56-57.

Reply:
Thanks! Fixed